.github/ISSUE_TEMPLATE/technical-debt.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Technical Debt
+about: Template for proposing solutions to technical debts
+title: ''
+labels: refactor
+assignees: ''
+
+---
+
+## Context
+
+What is the problem that you are trying to solve?
+Why is this problem relevant?
+How this problem affects the feature roadmap?
+Is it a blocker to implement a new future?
+What parts of the architecture and code are affected?
+
+## Solution
+
+What are the possible solutions?
+If there are multiple, what are the benefits and drawbacks of each one?
```

Comment:
If there are multiple alternatives, what are the benefits and drawbacks of each one?